src/features/assets/data/circulating-supply.ts

```diff
@@ -0,0 +1,31 @@
+import algosdk from 'algosdk'
+import { atom } from 'jotai'
+import { getApplicationResultAtom } from '@/features/applications/data'
+import { fundedAccountAtom } from '@/features/accounts/data'
+import { algorandClient } from '@/features/common/data/algo-client'
+
+export const createAssetCirculatingSupplyAtom = (applicationId: bigint, assetId: bigint) => {
```

Comment:
Extracted this into an atom creator, rather than using our atom cache. The atom cache isn't needed here as the asset metadata itself is being cached, so using it meant that data was cached twice.